_overviews/hello-scala/getting-started.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: multipage-overview
+title: Getting Started
+description: How to get started working with Scala.
+partof: hello_scala
+overview-name: Hello, Scala
+discourse: true
+num: 3
+outof: 54
+previous-page: prelude-taste-of-scala
+next-page: scala-features
+---
+
+
+In this book we assume that you’re familiar with another language like Java, so we don’t spend much time on programming basics. That is, we assume that you’ve seen things like for-loops, classes, and methods before, so we generally only write, “This is how you create a class in Scala,” that sort of thing.
+
+That being said, there are a few good things to know before you read this book.
+
+
+
+## Download Scala
+
+First, to run the examples in this book you’ll need to download the Scala compiler and runtime environment. You can download it from this URL:
+
+- [scala-lang.org/download/](https://www.scala-lang.org/download/)
```

Comment:
I don’t know if you’ll like these changes, but I added the link to the Getting Started page, then changed this page’s name from “Getting Started” to “Preliminaries.” It definitely looked wrong to go from one Getting Started page to another Getting Started page.

Also, I don’t know if it’s a good idea, but I open the Getting Started link in a new browser window. My thinking is that because the book portion of the website looks like the non-book portion of the website, it can be confusing to leave the book for other pages on the website. Or I should say that I thought it might be confusing because we go from a page with a ToC on the right to a similar-looking page with a different ToC on the right.

Maybe it would help if the ToC title was “Book Contents,” or if there’s some way to show a book icon or something, but that’s what I was thinking. (Or it may just be me that thinks this is a little confusing.)

Reply:
I love the new structure! However, I prefer having a regular link rather than a link that opens a new window, do you mind reverting that change?

Reply:
Okay, I changed that link to be a normal Markdown link, and also renamed that page from *getting-started.md* to *preliminaries.md*.

Later last night I thought that maybe something that could be done (some time in the future) is to highlight links that go to pages out of books a little like how they do on Wikipedia, adding a little image next to the link:

![external-link](https://user-images.githubusercontent.com/309803/66332709-8dc8fd80-e8f2-11e9-9a40-7d49a256b857.jpg)

I have several of my books on my website, and I think this might help in my own case.